### PR TITLE
CNV - sriov #26628 - live migration not supported for VMs with sriov 

### PR DESCRIPTION
**Pull request by aburdenthehand**
Adding note to livemigration and sriov concepts re: live migration not supported for VMs attached to sriov interfaces. Also a random typo
#26628

### CODE REVIEW

**Comment by yossisegev** [comment]
LGTM

File: modules/virt-understanding-live-migration.adoc [hunk 1]
```diff
@@ -17,3 +17,8 @@ which it is running is placed into maintenance.
 Virtual machines must have a persistent volume claim (PVC)
 with a shared ReadWriteMany (RWX) access mode to be live migrated.
 ====
+
```
Comment (ousleyp) [inline modules/virt-understanding-live-migration.adoc:20]: My only observation is that it's a little weird to have two notes in a row, and I can't tell if the first note really needs to be designated as more important than the second note.

**Review by ousleyp — APPROVED** [review]
LGTM, just one observation.

**Pull request merged**